site2/website-next/versioned_docs/version-2.7.3/deploy-bare-metal.md

```diff
@@ -243,7 +243,7 @@ $ PULSAR_EXTRA_OPTS="-Dstats_server_port=8001" bin/pulsar-daemon start zookeeper
 
 Once you deploy ZooKeeper for your cluster, you need to write some metadata to ZooKeeper for each cluster in your instance. You only need to write this data **once**.
 
-You can initialize this metadata using the [`initialize-cluster-metadata`](reference-cli-tools.md#pulsar-initialize-cluster-metadata) command of the [`pulsar`](reference-cli-tools.md#pulsar) CLI tool. This command can be run on any machine in your ZooKeeper cluster. The following is an example:
+You can initialize this metadata using the [`initialize-cluster-metadata`](reference-cli-tools.md#pulsar-initialize-cluster-metadata) command of the [`pulsar`](reference-cli-tools.md#pulsar) CLI tool. This command can be run on any machine in your Pulsar, so metadata can be initialized from a ZooKeeper, Broker, or Bookie machine. The following is an example:
```

Comment:
@frankxieke can you review it from the technical perspective? Thanks

Reply:
I think the original doc meaning is "this command can be run on any machine of ZooKeeper cluster", because one of the argument of the following command is zookeeper address.

Reply:
There is some misunderstanding here. 
I think the original doc meaning is "this command can be run on any machine, and the zookeeper argument can be any machine of Zookeeper cluster", because one of the argument of the following command is zookeeper address.